### PR TITLE
BAEL-973 Apache Commons Commons IO

### PR DESCRIPTION
**Pull request by charithdesilva**
@kevin - Changes were made as per your request. Thanks.

### CODE REVIEW

File: libraries/src/main/java/com/baeldung/commons/io/FileMonitor.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import org.apache.commons.io.monitor.FileAlterationListener;
+import org.apache.commons.io.monitor.FileAlterationListenerAdaptor;
+import org.apache.commons.io.monitor.FileAlterationMonitor;
+import org.apache.commons.io.monitor.FileAlterationObserver;
+
+import java.io.File;
+
+public class FileMonitor {
+
+	public static void main(String[] args) throws Exception {
```
Comment (KevinGilmore) [inline libraries/src/main/java/com/baeldung/commons/io/FileMonitor.java]: Need to use spaces (not tabs) for indentation. There's a standard formatter for Eclipse that you can import -- see the "eclipse" folder of the tutorials repo.

**Comment by charithdesilva** [comment]
I'll check this tonight. Thanks 👍

**Comment by charithdesilva** [comment]
@KevinGilmore formatted using formatter.xml. Thanks

File: libraries/src/test/java/com/baeldung/commons/io/CommonsIOUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        FileUtils.copyFileToDirectory(file, tempDir);
+        File newTempFile = FileUtils.getFile(tempDir, file.getName());
+        String data = FileUtils.readFileToString(newTempFile,
+            Charset.defaultCharset());
+
+        Assert.assertEquals(expectedData, data.trim());
+    }
+
+    @Test
+    public void whenUsingFileNameUtils_showdifferentFileOperations()
```
Comment (KevinGilmore) [inline libraries/src/test/java/com/baeldung/commons/io/CommonsIOUnitTest.java]: missing the "then" part of the test name

File: libraries/src/test/java/com/baeldung/commons/io/CommonsIOUnitTest.java [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+        new TeeInputStream(new ByteArrayInputStream(str.getBytes()),
+            new TeeOutputStream(outputStream1, outputStream2), true)
+                .read(new byte[str.length()]);
+
+        Assert.assertEquals(str, String.valueOf(outputStream1));
+        Assert.assertEquals(str, String.valueOf(outputStream2));
+    }
+
+    @Test
+    public void whenGetFilewithNameFileFilter_then_FindfileTesttxt()
```
Comment (KevinGilmore) [inline libraries/src/test/java/com/baeldung/commons/io/CommonsIOUnitTest.java]: Don't need underscore after the word "then"

**Pull request merged**